reference/email.md

```diff
@@ -1,25 +1,28 @@
 # Sending E-Mail
 
-By default only the master environment can send emails and there is no
-need to additionally configure your web application to enable that. For
-the non-master environment this feature can be enabled by using the
-Platform CLI.
+By default only the master environment can send emails.
+For the non-master environments, you can configure outgoing emails via the
+[Web Interface](../overview/web-ui/configure-environment.html#settings).
 
-For example, in case you want to enable sending emails for your
-non-master environment use this command:
+Emails from Platform.sh are sent via a SendGrid-based SMTP proxy.
+Each Platform.sh project is provisioned as a SendGrid sub-account.
+These SendGrid sub-accounts are capped at 12k emails per month.
+You can use `/usr/sbin/sendmail` on your application container to send emails
+with the assigned SendGrid sub-account.
 
-```
-$ platform environment:metadata enable_smtp true
-```
+Alternatively, you may use your own SMTP server.
+In this case, use port 465 / 587 since port 25 is blocked for security reasons.
+
+> **note**:
+> If you need to adopt SPF for improving email deliverability, please read
+> [SPF setup guidelines on SendGrid](https://sendgrid.com/docs/Glossary/spf.html).
 
-Emails from Platform.sh are sent via a SendGrid-based SMTP proxy. Each
-Platform.sh project is provisioned as a SendGrid sub-account. Note that SendGrid
-sub-accounts are capped at 12k emails per month. If you need to adopt SPF for
-improving email deliverability, please read
-[SPF setup guidelines on SendGrid](https://sendgrid.com/docs/Glossary/spf.html).
+## Sending email in PHP
 
-When you send email you should use the `mail` facility. There is no SMTP 
-server running on localhost. For example in Symfony, if you use the default `SwiftMailer`,
+When you send email, you should use the built-in `mail` facility in PHP.
```

Comment:
Let's say "the build-in `mail()` function in PHP".  No room for confusion that way.